wc-i18n.html

```diff
@@ -182,7 +189,12 @@
      * Get the translation string and replace the content of the element.
      */
     update: function(locale) {
-      var msg = this._domains[this._domain].locales[locale][this._msgid];
+      var locales = this._domains[this._domain].locales;
+      if(!locales[locale] || !locales[locale][this._msgid]) {
+        console.warn('wc-i18n: Language/Key combination not found: ' + locale + ' ' + this._msgid);
+        return this.fire('wc-i18n:locale-not-found', {locale: locale, key: this._msgid});
```

Comment:
In `v2` of this component, the error event here has been refactored to `wc-i18n-translations-error`. If you want to update this to that name, then there is less refactoring that needs to happen from `v1` -> `v2`. Do that and this is good to go.